clarity_ext/domain/artifact.py

```diff
@@ -0,0 +1,49 @@
+from clarity_ext.domain import Analyte
+from clarity_ext.domain import Container
+
+
```

Comment:
Maybe just StepRepository? Or StepArtifactRepostory? The good thing with the current name is that I know exactly what it is as long as I have druggled though the Clarity XML replresentation, But without a deep dive into the XML the 'InputOutputMap' says not very much at all.